js/renovation/viz/core/common/types.d.ts

```diff
@@ -1,4 +1,4 @@
-export type Canvas = { left?: number; top?: number; right?: number; bottom?: number; width?: number; height?: number };
+export type Canvas = { left: number; top: number; right: number; bottom: number; width: number; height: number };
```

Comment:
It similar to the predefined `ClientRect` interface. Should we add our own Canvas? Maybe just Canvas = ClienttRect or use ClientRect instead of Canvas?

```ts
interface ClientRect {
    bottom: number;
    readonly height: number;
    left: number;
    right: number;
    top: number;
    readonly width: number;
}
```

Reply:
Fixed